assets/src/application/ui/forms/ee-form/RenderSection.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { SectionProps } from './types';
+import RenderFields from './RenderFields';
+
+const RenderSection: React.FC<SectionProps> = ({ name, title, fields, namespaceFields }) => {
```

Comment:
Maybe smth different than `namespaceFields` ? :smiley:

Reply:
May be our CNO (@tn3rb) has a better idea 😄